ghcide/src/Development/IDE/Spans/Documentation.hs

```diff
@@ -67,7 +67,7 @@ getDocumentationTryGhc env mod n = head <$> getDocumentationsTryGhc env mod [n]
 
 getDocumentationsTryGhc :: HscEnv -> Module -> [Name] -> IO [SpanDoc]
 getDocumentationsTryGhc env mod names = do
-  res <- catchSrcErrors (hsc_dflags env) "docs" $ getDocsBatch env mod names
```

Comment:
For the current commit, I do keep external API unchanged, since the main idea is to provide proper data type there first.